packages/components/src/components/SquiggleViewer/VariableBox.tsx

```diff
@@ -66,6 +81,7 @@ export const VariableBox: React.FC<VariableBoxProps> = ({
               />
             </span>
             <span className="text-stone-800 font-mono text-sm">{name}</span>
+            {preview && <div className="ml-2">{preview}</div>}
```

Comment:
Btw, why are we hiding the header (right side text) when the item is collapsed?

It's weird that:
- when the item is expanded, there is duplicate info about lists and records
- when other types of items are collapsed, there's no data about the type

Reply:
Agreed it's a bit weird. 

I think arrays and records are particularly good for the preview, because a small bit of information goes a long way. Later on, once we have icons, we can show the other items. 

There's a bit of duplicate information, but not too much. The right content then is has the full name.